pyproject.toml

```diff
@@ -96,3 +96,33 @@ ignore = [
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tool.pixi.dependencies]
+gymnasium = ">=0.29.1"
+loop-rate-limiters = ">=1.0.0"
+msgpack-python = ">=1.0.2"
+numpy = ">=1.24.3"
+scipy = ">=1.10.0"
+upkie_description = ">=2.1.0"
+
+[tool.pixi.pypi-dependencies]
+upkie = { path = ".", editable = true }
+
+[tool.pixi.feature.docs.dependencies]
+python = "=3.12"
+doxygen = "<=1.9.5"
+gymnasium = "*"
+loop-rate-limiters = "*"
+msgpack-python = "*"
+numpy = "*"
+pinocchio = "*"
+scipy = "*"
+stable-baselines3 = "*"
+upkie_description = "*"
```

Comment:
Since the doc environment will also use the default feature, you don't have to repeat the default feature dependencies:
- gymnasium
- loop-rate-limiters
- msgpack-python
- numpy
- scipy
- upkie_description